.github/workflows/release.yml

```diff
@@ -4,9 +4,14 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'The version that is going to be released. Should be in format 7.y.z'
+        description: 'The version that is going to be released. Should be in format 7.yy.z'
+        required: true
+      dwoVersion:
+        description: 'The version of DevWorkspace Controller. Should be in format v0.y.z'
```

Comment:
why not compute these values from `version` like you did using the chectl version?

Reply:
I did this as a temporary hack for the workflow, that is triggered automatically after che operator release completion (PR merge). All we can get from it now, is the version in the VERSION file, and DWCO while is released with Che operator, is is not present anywhere else in the source code, so I decided to compute it that way. 
In future (was expecting it for 7.29.0 but it got delayed) DWCO version will not be required for chectl release

Reply:
hmm. ok.